docs/user/supported_devices.rst

```diff
@@ -500,6 +504,11 @@ Footnotes
 .. [#lan_as_wan]
   All LAN ports on this device are used as WAN.
 
+.. [#missing_radios]
+  These devices will not make use of all available radios,
+  as triple- or n-radio setups are just not supported in gluon, yet.
+  All radios are working though; setting them up manually could be an (unstable) option.
```

Comment:
I don't think we should mention unstable options in the docs. I'd keep this note a bit shorter, for example:

```suggestion
  This device contains more than two WLAN radios, which is currently
  unsupported by Gluon. Only the first two radios will work.
```